internal/schema/stacks/1.8/output_block.go

```diff
@@ -36,7 +38,18 @@ func outputBlockSchema() *schema.BlockSchema {
 				"type": {
 					Constraint:  schema.AnyExpression{OfType: cty.DynamicPseudoType},
 					IsRequired:  true,
-					Description: lang.PlainText(""), // TODO
+					Description: lang.PlainText("Type of the output value"),
+				},
+				"sensitive": {
+					Constraint:   schema.LiteralType{Type: cty.Bool},
+					DefaultValue: schema.DefaultValue{Value: cty.False},
+					IsOptional:   true,
+					Description:  lang.PlainText("Whether the output contains sensitive material and should be hidden in the UI"),
+				},
+				"ephemeral": {
+					Constraint:  schema.LiteralType{Type: cty.Bool},
+					IsOptional:  true,
+					Description: lang.PlainText("Whether the output is ephemeral and should not be persisted"),
```

Comment:
`sensitive` and `ephemeral` were just added to Stacks 3 weeks ago and I think they shouldn't be part of this 1.8 schema.

Reply:
Fixed!